internal/action/generate.go

```diff
@@ -220,7 +220,7 @@ func clamp(min, max, value int) int {
 }
 
 func (s *Action) generatePasswordForRule(ctx context.Context, c *cli.Context, length, name, domain string, rule pwrules.Rule) (string, error) {
-	out.Printf(ctx, "Using password rules for %s ...", domain)
+	out.Noticef(ctx, "Using password rules for %s ...", domain)
```

Comment:
Maybe rephrase that to make it more obvious it's due to the domain name?
Something like :
"The domain %s appears to have specific password rules, using them."